style.css

```diff
@@ -211,37 +206,47 @@ table.affiliations {
 
 
 code {
-	font-family: Consolas,"courier new";
+	font-family: Consolas, "courier new";
 	color: crimson;
 	background-color: #f1f1f1;
 	padding: 2px;
 	font-size: 105%;
-	}
+}
 
-.sensitive{
-display: block;
+.sensitive {
+	display: block;
 	text-align: center;
 	margin: auto;
 }
 
 .block {
-    display: block;
-  }
- 
-.warning-overlay {
-    position: absolute;
-    top: 0;
-    left: 0;
-    width: 100%;
-    height: 100%;
-    background-color: rgba(0, 0, 0, 0.5); /* Semi-transparent black overlay */
-    color: white;
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    font-size: 24px;
-    font-weight: bold;
-    text-align: center;
-    cursor: pointer;
-  }
+	display: block;
+}
+
 
+.nsfw-img-container {
+	position: relative;
+}
+
+/* the overlay  */
+.nsfw-img-container :first-child {
+	position: absolute;
+	top: 0;
+	left: 0;
+	width: 100%;
+	height: 100%;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	font-size: 24px;
+	font-weight: bold;
+	cursor: pointer;
+	backdrop-filter: blur(14px);
+	color: white;
+	background-color: #8080807a;
```

Comment:
tweak `blur(14px)` and the backround-color (including transparancy) to adjust effects.